util/regtool.py

```diff
@@ -192,6 +196,18 @@ def main():
         log.error(str(err))
         exit(1)
 
+    # If this block has countermeasures, we grep for RTL annotations in all
+    # .sv implementation files and check whether they match up with what is
+    # defined inside the Hjson.
+    annotation_list = []
+    if obj.countermeasures:
```

Comment:
Are we ok with running this always, or should we only run this in certain modes (e.g. reggen rtl generation mode)?

Reply:
I think we should run it unconditionally. I also don't think that it should check whether `obj.countermeasures` is nonempty.

Reply:
Ok good I'll remove that.

Reply:
For RV_DM, I am adding a custom Hjson file to capture the debug mem region, which is a part of external 3rd party code.This is explicitly only for DV and is not meant to contain any countermeasures. See comment here: 

https://github.com/lowRISC/opentitan/pull/10858/commits/64d6c581bffc06de0cadf78d0977d98fbf7eb86b#r807576111

I can condition this on whether we are dumping DV code.